x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/view/details/components/endpoint_details_tabs.tsx

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { memo, useCallback, useMemo, useState } from 'react';
+import styled from 'styled-components';
+import { EuiTabbedContent, EuiTabbedContentTab } from '@elastic/eui';
+import { EndpointIndexUIQueryParams } from '../../../types';
+export enum EndpointDetailsTabsTypes {
+  overview = 'overview',
+  activityLog = 'activity-log',
+}
+
+export type EndpointDetailsTabsId =
+  | EndpointDetailsTabsTypes.overview
+  | EndpointDetailsTabsTypes.activityLog;
+
+interface EndpointDetailsTabs {
+  id: string;
+  name: string;
+  content: JSX.Element;
+}
+
+const StyledEuiTabbedContent = styled(EuiTabbedContent)`
+  overflow: hidden;
+  padding-bottom: 48px;
```

Comment:
We should not be using hard coded px values (or colors), but rather grabbing these values from the `theme` variable that is available when you defined the value here as a code function in the string template. Example:

```suggestion
  padding-bottom: {(props) => props.theme.eui.paddingSizes.xl};
```

In this case, it seems as if you need a padding value that might not exist. I would work with Bonnie to see if we can keep to defined sizes (xs, s, m, l, xl). If not then this might work (never tried it):

```
  padding-bottom: calc( {(props) => props.theme.eui.paddingSizes.l} + {(props) => props.theme.eui.paddingSizes.l}  );
```

styled components might also already provide some calc like functions - not sure.

Reply:
FYI - a while back I created this gist for my use in trying to understand what's available for use in the `theme.eui`:

https://gist.github.com/paul-tavares/303d718ac7f514d6ee681a3851407a7e

Reply:
I agree that hardcoded values are not maintainable. I'm quite sure this won't be needed when we refactor the tabs component to use the [flyout#more-complicated-flyout](https://elastic.github.io/eui/#/layout/flyout#more-complicated-flyout) component version. I'll change this to use the `xl` padding for now.

Reply:
done 31cf7af4edb6471cf953c2df1fdeea5cfd61f65a